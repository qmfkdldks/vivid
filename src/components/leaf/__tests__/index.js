import React from "react";
import { shallow } from "enzyme";
import Leaf from "../index";

describe("Leaf", () => {
  it("should render successfully", () => {
    const component = shallow(<Leaf />);
    expect(component).toMatchSnapshot();
  });
});
