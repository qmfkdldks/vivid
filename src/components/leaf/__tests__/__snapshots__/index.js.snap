// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Leaf should render successfully 1`] = `<span />`;
